frontend/src/lib-customizations/espoo/employee-mobile-frontend/assets/i18n/fi.ts

```diff
@@ -90,12 +90,11 @@ export const fi = {
   absences: {
     title: 'Poissaolomerkintä',
     absenceTypes: {
-      OTHER_ABSENCE: 'Muu poissaolo',
+      OTHER_ABSENCE: 'Poissaolo',
       SICKLEAVE: 'Sairaus',
       UNKNOWN_ABSENCE: 'Ilmoittamaton poissaolo',
-      PLANNED_ABSENCE: 'Suunniteltu poissaolo / vuorohoito',
+      PLANNED_ABSENCE: 'Vuorotyöpoissaolo',
```

Comment:
`PLANNED_ABSENCE` now has varying labels. Should the citizen labels be updated accordingly?
<img width="378" alt="image" src="https://user-images.githubusercontent.com/1176169/156117831-44ebbeb1-a994-4264-89b5-c8746ceedea7.png">

Reply:
The labels have now been updated and the option has been hidden altogether if none of the children are in shift care.